### PR TITLE
TST: changed behavior of df.loc on multiindex

### PR DESCRIPTION
**Pull request by gamenerd457**
-added unittest  for issue #43599 
-error occurs due to different versions of pandas

### CODE REVIEW

**Review by mzeitlin11 — CHANGES_REQUESTED** [review]
Thanks for the pr @gamenerd457.  Please look at the contributing guide (https://pandas.pydata.org/docs/development/index.html#development), especially the portions about testing (we don't use `unittest`) and style (this is currently failing checks)

File: pandas/tests/indexing/test_loc.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
 from pandas.api.types import is_scalar
 from pandas.core.api import Float64Index
 from pandas.tests.indexing.common import Base
+import unittest
```
Comment (phofl) [inline pandas/tests/indexing/test_loc.py:41]: We use pytest

**Review by phofl — CHANGES_REQUESTED** [review]
Please use pre-commit

**Comment by gamenerd457** [comment]
> Thanks for the pr @gamenerd457. Please look at the contributing guide (https://pandas.pydata.org/docs/development/index.html#development), especially the portions about testing (we don't use `unittest`) and style (this is currently failing checks)

I have made the changes

**Review by jreback — CHANGES_REQUESTED** [review]

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by jreback** [comment]
ca you merge master and update to comments

**Comment by mroeschke** [comment]
Thanks for the PR but appears to have gone stale. If interested in continuing please merge the main branch, address the review, and we can reopen.

**Pull request closed without merging**